simulation/tools/CMakeLists.txt

```diff
@@ -1,31 +1,38 @@
 set(SUBSYS_NAME tools)
 
-find_package(GLEW)
 find_package(GLUT)
+find_package(GLEW CONFIG)
```

Comment:
Do you need to do this? Isn't tools/CMakeLists.txt included from Simulation main CMakeLists, where we just searched for GLEW?